cubecl-book/src/core-features/features.md

```diff
@@ -0,0 +1,52 @@
+# Querying hardware features
+
+Some features and datatypes are only supported on some hardware or some backends. They can be
+queried with:
+
+```rust, ignore
+client.properties().feature_enabled(feature)
+```
+
+Also see [`Feature`](https://docs.rs/cubecl/latest/cubecl/enum.Feature.html).
+
+## Overview
+
+### Features
+
+Also requires device support
+
+| Feature | CUDA | ROCm | WGPU (WGSL) | WGPU (SPIR-V) |
+| ------- | ---- | ---- | ----------- | ------------- |
+| Subcube | ✔️   | ✔️   | ✔️          | ✔️            |
```

Comment:
There is a format error in the table.

Reply:
That's actually technically correct, it formats based on character count, but emojis are displayed as wider than normal characters. So the formatter automatically changes it to this.